<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1331</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Lista Simple</string>
     </attribute>
     <widget class="QGraphicsView" name="VistaLista">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>1071</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="TextoIngresar_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>541</height>
       </rect>
      </property>
      <property name="title">
       <string>ListaSimple</string>
      </property>
      <widget class="QLineEdit" name="TextoIngresar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Boton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingresar un valor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Valor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar Valor</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TextoEliminar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonEliminar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TextoInsertar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>InsertarPorPosicion</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonInsertar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Insertar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar Elemento</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TextoBuscar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonBuscar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="GuardarListaSimple">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CargarListaSimple">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Pila</string>
     </attribute>
     <widget class="QGraphicsView" name="VistaPila">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>1011</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="TextoIngresar_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>121</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Pila</string>
      </property>
      <widget class="QLineEdit" name="TextoApilar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonIngresarPila">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Apilar Valor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Apilar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Desapilar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonEliminarPila">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Tope</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonMirarTope">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Mirar tope</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar En Pila</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BuscarPila">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonBuscarPila">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="GuardarPila">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CargarPila">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Cola</string>
     </attribute>
     <widget class="QGroupBox" name="TextoIngresar_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Cola</string>
      </property>
      <widget class="QLineEdit" name="TextoEncolar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonEncolar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Encolar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::PreventContextMenu</enum>
       </property>
       <property name="text">
        <string>Encolar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Desencolar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonDesencolar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Frente</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonMirarFrente">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Mirar Frente</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar En Cola</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BuscarCola">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonBuscarCola">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="GuardarCola">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CargarCola">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar</string>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="VistaCola">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>1001</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Lista doble</string>
     </attribute>
     <widget class="QGroupBox" name="TextoIngresar_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>121</width>
        <height>541</height>
       </rect>
      </property>
      <property name="title">
       <string>ListaSimple</string>
      </property>
      <widget class="QLineEdit" name="TextoIngresarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonIngresarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingresar un valor</string>
       </property>
      </widget>
      <widget class="QLabel" name="Valor">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Valor</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar Valor</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TextoEliminarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonEliminarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TextoInsertarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>InsertarPorPosicion</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonInsertarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Insertar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar Elemento</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="TextoBuscarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BotonBuscarDLL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="GuardarListaDoble">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="CargarListaDoble">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar</string>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="VistaLista_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>1071</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1353</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
